### PR TITLE
[CMS-25724] Replace deprecated ArchiveProcess in scheduled publication workflows definition 

### PR DESCRIPTION
**Pull request by dahoeck**
removel already happend in https://github.com/CoreMedia/coremedia-additional-workflows/commit/050a13bbda40f4c7eac6d4e429333c0f4b1c44bd

### CODE REVIEW

**Review by ahubold — CHANGES_REQUESTED** [review]
You can't simply remove the whole "Finish" task. It's still needed as final step, and it is also referenced from previous task(s). You'll probably run into an error when trying to upload this one.

You can remove the finishTaskRetryTimer variable, the TimerHandler, the AssignVariable for finishTaskRetryTimer and the EnableTimer action.

The Finish task itself must be kept. Also the assignment of "changeSetLockedInStudio" should probably be kept. It's at least unrelated to archiving, even though it might not really make a difference (compare CMS-19032).

Compare to other workflow definitions. It should look similar at the end: https://github.com/CoreMedia/coremedia-additional-workflows/blob/main/two-step-publication-workflow/two-step-publication-workflow-server/src/main/resources/com/coremedia/blueprint/workflow/extended-studio-two-step-publication.xml#L185-L193

**Comment by dahoeck** [comment]
My bad. Misread the diff to the other workflow definition. I've now kept the Finish task in place and only removed the timer assignment

File: scheduled-publication-workflow/scheduled-publication-workflow-server/src/main/resources/com/coremedia/blueprint/scheduledpublication/studio-scheduled-publication.xml [hunk 1]
```diff
@@ -143,19 +143,15 @@
...
-
     <AutomatedTask name="Finish" final="true">
       <Action class="AssignVariable" resultVariable="changeSetLockedInStudio">
         <Boolean value="false"/>
       </Action>
-      <Action class="AssignVariable" resultVariable="finishTaskRetryTimer">
-        <Timer value="30"/>
-      </Action>
-      <Action class="EnableTimer" timerVariable="finishTaskRetryTimer"/>
+      <Action class="ArchiveProcess" maxProcessesPerUser="100"/>
```
Comment (ahubold) [inline scheduled-publication-workflow/scheduled-publication-workflow-server/src/main/resources/com/coremedia/blueprint/scheduledpublication/studio-scheduled-publication.xml]: You have re-introduced the deprecated ArchiveProcesss action here. It doesn't do anything and might be deleted in the future. You should remove that line from all workflow definitions.

**Review by ahubold — CHANGES_REQUESTED** [review]

**Review by ahubold — APPROVED** [review]
:+1: LGTM

**Pull request merged**